service-directory/Readme.md

```diff
@@ -6,15 +6,45 @@ It is preconfigured to use a local running [Ganache](https://www.trufflesuite.co
 
 ## Prerequisities
 
-1. Installation of [Ganache](https://www.trufflesuite.com/docs/ganache/quickstart)
+1. Installation of ganache CLI as described [here](https://github.com/trufflesuite/ganache#command-line-use). Alternatevily installation of [Ganache](https://www.trufflesuite.com/docs/ganache/quickstart). CLI is the preferred way for the scripts.
 2. Installation of [Truffle](https://www.trufflesuite.com/docs/truffle/getting-started/installation)
 
-## Using it with truffle
+## Using ganache-cli with provided scripts
+
+### Starting ganache
+
+The provided script [start-ganache-aea.sh](start-ganache-aea.sh) starts a local ganache-cli instance with the following parameters:
+
+- network id 1111, which will be used later to identify the network to retrieve the deployed contract address
+- Running port of 8545
+- Proper mnemonic configuration for the preconfigured accounts in the agents
+
+This configuration is also reflected in the [truffle-config.js](truffle-config.js) file in the *development* network.
+
+### Deploying contracts to ganache
+
+Migrating/deploying to Ganache can be done using `truffle migrate`.
+
+### Updating contract address in agents
+
+As the agents needs to know the current address of the newly deployed [*ServiceDirectory*](contracts/ServiceDirectory.sol), this address needs to be updated in the corresponding *skill.yaml* files.
```

Comment:
Should we detail, where to find that contract address?

Reply:
Just found the update-contract-address.sh. This should definitely be mentioned.

Reply:
I see...
The idea was to describe how to update them in a manual way and then point to the script, which is described at line 40.
Maybe I have to reorder it to mention first the script and then how to do it in a manual way for custom configuration?

Reply:
Ah i see. I think in a rendered markdown that's clear. I got distracted by the code block and lost context, when reading line 40.

Reply:
I will reorganize it a little bit